ModuleOneFirstTaskConsole/ModuleOneFirstTaskConsole.csproj

```diff
@@ -14,4 +14,8 @@
     </PackageReference>
   </ItemGroup>
```

Comment:
Please don't forget to include `StyleCop.Analyzers` in all projects.

Reply:
I hope I will. Every project has a reference, as I see. Thank you!
![image](https://user-images.githubusercontent.com/42671551/113203073-3b149b80-9274-11eb-86ac-b75691ccd1ef.png)